xarray/tests/test_computation.py

```diff
@@ -784,6 +781,10 @@ def test_dot(use_dask):
     assert (actual.data == np.einsum('ij,ijk->k', a, b)).all()
     assert isinstance(actual.variable.data, type(da_a.variable.data))
 
+    import dask
```

Comment:
This needs to be guarded behind `use_dask`. Currently the tests without dask installed are failing here on Travis-CI.